Add more icons

pinussilvestrus opened the pull request:
Closes #13 

This PR
* adds all element icons
* removes the markers for now
* displays the header icon into a 32x32 viewbox

pinussilvestrus commented:
Don't know what's wrong with Codecov here...

pinussilvestrus commented on src/bpmn-properties-panel/icons/index.js:
```diff
@@ -1,22 +1,175 @@
-import StartEventIcon from './StartEvent.svg';
-import SubProcessCollapsedIcon from './SubProcessCollapsed.svg';
-import UserTaskIcon from './UserTask.svg';
+import BusinessRuleTaskIcon from './bpmn-icon-business-rule-task.svg';
```
This file is really blown up by the number of icons. For now, I can't see real alternatives, because we have to very explicit.

andreasgeier reviewed (changes requested):
A lot of icons do not show up in the header: transaction, all boundary events, all data-related elements, pool ...

pinussilvestrus commented:
I'll double-check. Probably some spelling mistakes with the types.

pinussilvestrus commented:
I updated the identifiers to show the correct icons.

pinussilvestrus commented:
> I updated the identifiers to show the correct icons. I also removed the legacy "default" icon, which was the old process icon. For now, nothing will be displayed if no icon was found.
> 
> The only missing icon seems to be for `Collaboration` though.

I added the missing icon, thanks to @andreasgeier (cf. https://github.com/camunda/camunda-modeler-design/issues/21)

andreasgeier reviewed (approved):
✅ &nbsp; All icons seem to be there now - good work!

andreasgeier commented:
I found a functional issue. When dropping the first pool to the canvas, the participant is selected but the properties panel shows the collaboration props. This happens only for the first pool, additional pools will show up in the panel correctly.

pinussilvestrus commented:
Let's [open an issue](https://github.com/bpmn-io/properties-panel-next/issues/new?assignees=&labels=bug&template=BUG_REPORT.md) for that 👍

andreasgeier commented:
Conditional flow icon is not showing up

andreasgeier commented:
default flow is missing as well

pinussilvestrus commented:
With https://github.com/bpmn-io/properties-panel-next/pull/56/commits/b204eb98b6be36f2603fb96030771c35d04e4e0f I added support for conditional + default flows. The icons were already present, just the detection failed.

![image](https://user-images.githubusercontent.com/9433996/121332206-ed717a80-c917-11eb-9916-a04d065b5033.png)

![image](https://user-images.githubusercontent.com/9433996/121332274-ff531d80-c917-11eb-99e5-3075d7739bc4.png)

----

Should be ready to review + merge now @barmac @MaxTru @andreasgeier. To not blow up this PR even further, let's consider open issues for further improvements (as missing icons).

barmac commented:
Is this expected that the icon changes its position for a super-long name of the element?

![image](https://user-images.githubusercontent.com/28307541/121357611-39311d80-c932-11eb-8c5a-d537c3319a02.png)

![image](https://user-images.githubusercontent.com/28307541/121357666-44844900-c932-11eb-97cd-aeaef88c5ad8.png)

pinussilvestrus commented:
Probably not. I fixed that to ensure the display box of the icon is stable.

![image](https://user-images.githubusercontent.com/9433996/121359759-19025e00-c934-11eb-9fce-53ac8b45c516.png)

![image](https://user-images.githubusercontent.com/9433996/121359781-1f90d580-c934-11eb-9b05-82136955382d.png)
